src/dswx_sar/save_mgrs_tiles.py

```diff
@@ -139,34 +316,36 @@ def run(cfg):
     input_list = cfg.groups.input_file_group.input_file_path
     dswx_workflow = processing_cfg.dswx_workflow
     product_version = cfg.groups.product_path_group.product_version
+    static_layer_cfg = cfg.groups.static_ancillary_file_group
+    mgrs_db_path = static_layer_cfg.mgrs_database_file
+    mgrs_collection_db_path = static_layer_cfg.mgrs_collection_database_file
```

Comment:
In the future, please consider adding some messages to inform the user/log these runconfig parameters.